File: src/main/java/me/yoursole/ctf/DataFiles/Items/CheapApple.java
package me.yoursole.ctf.DataFiles.Items;

import me.yoursole.ctf.DataFiles.GameData;
import org.bukkit.*;
import org.bukkit.inventory.*;
import org.bukkit.inventory.meta.ItemMeta;
import org.bukkit.inventory.meta.SkullMeta;

import java.util.ArrayList;
import java.util.List;
import java.util.UUID;

public class CheapApple {
    public static ItemStack item;
    static {
        ItemStack apple = new ItemStack(Material.APPLE);
        ItemMeta meta = apple.getItemMeta();

        List<String> lore = new ArrayList<>();
        meta.setDisplayName(ChatColor.RED + "Cheap Apple");
        lore.add(ChatColor.GRAY + "For when you are feeling broke");
        meta.setLore(lore);
        apple.setItemMeta(meta);
        CheapApple.item = apple;
    }
}

File: src/main/kotlin/me/yoursole/ctf/datafiles/items/CheapApple.kt
package me.yoursole.ctf.datafiles.items

import me.yoursole.ctf.datafiles.Utils.setCTFId
import org.bukkit.Material
import org.bukkit.craftbukkit.v1_18_R2.inventory.CraftItemStack
import org.bukkit.enchantments.Enchantment
import org.bukkit.inventory.ItemStack

object CheapApple {
    val item: CraftItemStack by lazy {
        CraftItemStack.asCraftCopy(ItemStack(Material.APPLE)).apply {
            itemMeta = itemMeta.apply {
                setDisplayName("§aCheap Apple")
                lore = listOf("§7For when you are feeling broke")
                addEnchant(Enchantment.THORNS, 10, true)
            }
            setCTFId("cheapapple")
        }
    }
}